### ISSUE TITLE
make html: misleading status message

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#522]
When `C:\Python<Version>\Scripts`  is not in path, `make html` correctly points out that it cannot find the `sphinx-build` command, but prints out a misleading status message in the end:

```
[C:\path\to\project]make html

TCC: C:\path\to\project\make.bat [41]  Unknown command
"sphinx-build"

Build finished. The HTML pages are in _build/html.
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/522
- Originally reported by: Santoso Wijaya
- Originally created at: 2010-09-08T02:58:18.854

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-10-22 09:54:44+00:00_

Should be fixed in <<cset aee3d7618ae3f3c2346ccb097c7efad591e967f4>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:30:10+00:00_

Removing version: 1.0.3 (automated comment)
